sensu/plugins/check-neutron-l3-routers.py

```diff
@@ -67,6 +82,8 @@ def main():
         sys.exit(STATE_WARNING)
 
     for router in router_list['routers']:
+        if delay_check(router, now, args.delay_seconds):
```

Comment:
does router have created_at and updated_at ? i know router-show does but not sure if list_routers contains that info.. probably it does.. but wanted to check what u found in unit test.